src/renderer/src/components/SkipMigrationModal/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useEffect } from 'react'
+import { CircularProgress, Dialog, Typography, useTheme } from '@mui/material'
+import ErrorIcon from '@mui/icons-material/Error'
+import { Column, Row } from '../LayoutComponents'
+import { defineMessages, useIntl } from 'react-intl'
+import { Button } from '../Button'
+
+export const SkipMigrationModal = ({ open, onClose }: SkipMigrationModalModalProps) => {
+  const intl = useIntl()
+  const theme = useTheme()
+  const [skipped, setSkipped] = useState(false)
```

Comment:
this just needs to be reset to `false` when unmounted. Otherwise when you open it again, it starts in the skipped state